python/lsst/daf/butler/core/location.py

```diff
@@ -368,6 +382,21 @@ def pathInStore(self):
         """
         return self._path
 
+    @property
+    def netloc(self):
+        """The URI network location."""
+        return self._datastoreRootUri.netloc
+
+    @property
+    def relativeToPathRoot(self):
+        """Returns the path component of the URI relative to the network
+        location.
+
+        Effectively, this is the path property with posix separator stipped
+        from the left hand side of the path.
+        """
+        return self.path.lstrip("/")
```

Comment:
`path` returns os path for file and schemeless but posixpath for anything else.

Reply:
Check the new version. I'm not sure you'd like it. Too many conversions happening left and right. Any suggestions welcomed.